packages/db/src/core/integration/index.ts

```diff
@@ -113,6 +106,51 @@ function astroDBIntegration(): AstroIntegration {
 						server.restart();
 					}
 				});
+				// Wait for dev server log before showing "connected".
+				setTimeout(() => {
+					logger.info(
+						connectToStudio ? 'Connected to remote database.' : 'New local database created.'
+					);
+					if (connectToStudio) return;
+
+					const localSeedPaths = SEED_DEV_FILE_NAME.map(
+						(name) => new URL(name, getDbDirectoryUrl(root))
+					);
+					let seedInFlight = false;
+					// Load seed file on dev server startup.
+					if (seedFiles.get().length || localSeedPaths.find((path) => existsSync(path))) {
+						loadSeedModule();
+					}
+					const eagerReloadIntegrationSeedPaths = seedFiles
+						.get()
+						// Map integration seed paths to URLs, if possible.
+						// Module paths like `@example/seed` will be ignored
+						// from eager reloading.
+						.map((s) => (typeof s === 'string' && s.startsWith('.') ? new URL(s, root) : s))
+						.filter((s): s is URL => s instanceof URL);
```

Comment:
nit: you should flip them :)

Reply:
Ah, I see where you're coming from but we actually cannot flip these. The `map()` is used to convert relative path strings to URLs, then the `filter()` consolidates to all URL objects. If we put the `filter()` first, those relative path strings would get filtered out